next-manifest.js

```diff
@@ -51,7 +51,7 @@ async function inlineManifest(manifest, swFilePath, precachePath) {
   const originalSw = await readFile(swFilePath, 'utf8');
 
   // Prepend/inline newly generated precache manifest and remove import for old one.
-  const manifestImportRegex = new RegExp(`(,\s*)?"${precachePath}"`);
+  const manifestImportRegex = new RegExp(`(,\s*)?"${precachePath}"(,\s*)?`);
```

Comment:
This regex as is doesn't work with additional paths in front of the manifest filepath (what happens when injecting instead of generating). 2.1 and 2.2 in my comment https://github.com/hanford/next-offline/issues/60#issuecomment-427045410.

Reply:
@hanford this is a _stretched_ regexp that does the job: 

```
(,\s*(\r\n|\r|\n)\s*)?"precache-manifest\.[^.]*\.js"(,\s*)?
``` 

but we need to separate the `manifestDest` glob (precache-manifest.*.js) and a `manifestRegexp` (precache-manifest\.[^.]*\.js) as I tried to be smarter and reuse the glob for the regex, as it worked.

<img width="699" alt="screen shot 2018-10-31 at 20 58 19" src="https://user-images.githubusercontent.com/474603/47818323-bf832d00-dd4f-11e8-972c-bd41fcfce8a3.png">

Reply:
Ahh, I'm okay with this, wanna do a quick PR? and I can cut a release

Reply:
> @hanford this is a _stretched_ regexp that does the job:
> 
> ```
> (,\s*(\r\n|\r|\n)\s*)?"precache-manifest\.[^.]*\.js"(,\s*)?
> ```

Why adding `(\r\n|\r|\n)\s*`? 

Doesn't a single `\s*` already cover this?
```
\s* matches any whitespace character (equal to [\r\n\t\f\v ])
* Quantifier — Matches between zero and unlimited times, as many times as possible, giving back as needed (greedy)
```

It already matches one of `[\r\n\t\f\v ]` zero and unlimited times.  

Agreed that we need to seperate `manifestDest` and a new `manifestRegexp` option.

Reply:
Without it it will replace both commas on the last example. That ensures that we're only replacing the leading comma if it has a newline in between.

Reply:
Ohhh, didn't catch the place where it replaces stuff. My bad. Thanks for the explanation.

Reply:
@orditeck if you update the PR I'd be happy to merge 😬